backend/tournesol/views/user.py

```diff
@@ -23,3 +30,29 @@ def delete(self, request):
         logout(request)
         logger.info("User '%s' with email '%s' has been deleted.", user.username, user.email)
         return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+
+
+class UserDataDumpView(APIView):
+    permission_classes = [IsAuthenticated]
+
+    def get(self, request):
+        response = HttpResponse(content_type='text/csv')
+        response['Content-Disposition'] = 'attachment; filename="export.csv"'
```

Comment:
Don't we want to wrap the file in a .zip archive, to be able to include multiple .csv files in the future?

Reply:
I thought it my be nice to also have the possibility to download only the comparisons as a csv. 

Not I added a second route:
- /exports/comparisons : downloads a csv with comparisons
- /exports/all : downloads a zip archive (currently only comparisons in it)